src/components/CollapsedOpeningTimes.js

```diff
@@ -47,20 +45,20 @@ export default class CollapsedOpeningTimes extends React.Component {
 
     // flow:disable not supported yet
     static sampleProps = {
-        "One opening time": fixture(true,  [{
+        "One opening time": fixture(true, [{
             day: "Wednesday",
             open: "10:30:00",
             close: "15:00:00",
-        },]),
+        }]),
         "Two opening times": fixture(true, [{
             day: "Wednesday",
             open: "00:00:00",
             close: "24:00:00",
-        },{
+        }, {
             day: "Thursday",
             open: "00:00:00",
             close: "23:00:00",
-        },]),
+        }]),
     };
 
     render(): React.Element {
```

Comment:
Not part of this change but I thought daniel recently updated all these to be ReactElement ?